data/cncf.yaml

```diff
@@ -250,19 +281,23 @@
       url: https://github.com/cilium/ebpf
       check_sets:
         - code-lite
+        - docs
     - name: cilium
       url: https://github.com/cilium/cilium
       check_sets:
         - community
         - code
+        - docs
     - name: hubble
       url: https://github.com/cilium/hubble
       check_sets:
         - code-lite
+        - docs
     - name: tetragon
       url: https://github.com/cilium/tetragon
       check_sets:
         - code-lite
+        - docs
```

Comment:
The `docs` check set is a subset of the `code-lite` check set, so it isn't needed in this case. Please remove it.